server/server/Cargo.toml

```diff
@@ -48,6 +48,7 @@ regex = { workspace = true }
 actix-multipart = { workspace = true }
 futures-util = { workspace = true }
 serde_json = { workspace = true }
+extism = "1.5.0"
```

Comment:
can you use workspace pattern please like the other creates

Reply:
general rule is when more then one crate uses them, btw I was also hoping you would do the same for server crate

Reply:
Ah nice I missed the service crate extism import, thank you